integration-tests/bean-validator/src/main/java/org/apache/camel/quarkus/component/bean/validator/it/BeanValidatorResource.java

```diff
@@ -39,13 +40,36 @@
     @Inject
     ProducerTemplate producerTemplate;
 
-    @Path("/get/{manufactor}/{plate}")
+    @Inject
+    ValidatorFactoryCustomizer.MyMessageInterpolator messageInterpolator;
+
+    @Path("/get/xml/{manufactor}/{plate}")
     @GET
     @Produces(MediaType.TEXT_PLAIN)
-    public Response get(@PathParam("manufactor") String manufactor, @PathParam("plate") String plate) throws Exception {
-        LOG.info("bean-validator: " + manufactor + "/" + plate);
-        Car car = new Car(manufactor, plate);
-        Exchange out = producerTemplate.request("direct:start", e -> e.getMessage().setBody(car));
+    public Response getXml(@PathParam("manufactor") String manufactor, @PathParam("plate") String plate) throws Exception {
+        return get(new CarWithAnnotations(manufactor, plate), "start");
+    }
+
```

Comment:
So `getAnot()` and `getXml()` have exactly same definition ? Is there really a need for 2 methods then ?

Reply:
There is an error in the test code, one method should use `CarWithAnnotations` and the other `CarWithoutAnnotations`.
Thanks for noticing it! (current code does not verify one configuration option) 
I'll fix it.